src/app/_components/PageHeader.tsx

```diff
@@ -8,25 +8,84 @@ import {
   DescriptionText,
 } from '@/components/DescriptionText'
 import { Heading } from '@/components/Heading'
-import { Meta } from '@/components/Meta'
+import { Meta, type MetaDataType } from '@/components/Meta'
+import { NextStaticImage } from '@/components/NextStaticImage'
 import { SectionDivider } from '@/components/SectionDivider'
 
 import { type CTAProps } from '@/types/sharedProps/ctaType'
-import { type ImageProps } from '@/types/sharedProps/imageType'
+import {
+  type StaticImageProps,
+  type ImageProps,
```

Comment:
`ImageProps`, `StaticImageProps`, `DynamicImageProps`, `NextImageProps` 😅 ~ I'm getting slightly confused. Let's talk about this later!

Reply:
✅